react/components/Dropzone/index.js

```diff
@@ -156,6 +170,12 @@ Dropzone.propTypes = {
   icon: PropTypes.node,
   /** Allow multiple files drop */
   multiple: PropTypes.bool,
+  /** Set accepted file types. See [this MDN article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept) for more information. */
+  accept: PropTypes.string,
```

Comment:
I know in the `react-dropzone` component this prop is a union of `string|string[]`, but I wanted to keep it simple here.